modules/pel/peltool/comp_id.py

```diff
@@ -3,23 +3,30 @@
 import json
 
 componentIDs = {}
-
+basePathPresent = os.path.exists('/usr/share/phosphor-logging/pels/')
+overallCheckCompIDFilePath, importSuccess = True, True
+checkCompIDFilePath = [overallCheckCompIDFilePath, importSuccess, basePathPresent]
 
 def getCompIDFilePath(creatorID: str) -> str:
     """
     Returns the file path to look up the component ID in.
     The pel_registry module isn't available on the BMC,
     so just look in /usr/share/... if that module isn't present.
     """
-    try:
-        import pel_registry
-        file = pel_registry.get_comp_id_file_path(creatorID)
-    except ModuleNotFoundError:
+    file = '/usr/share/phosphor-logging/pels/'
```

Comment:
Is there a way you can get down to having 1 var hardcoding the /usr/share/ path?
e.g.
basePath = '/usr/share...'
file = basePath